helpers/useReviewSectionRows.tsx

```diff
@@ -377,6 +385,15 @@ const useReviewSectionRows = <
             []
           );
 
+          const changeSlug = repeatingStepSlugs.includes(row.changeSlug)
+            ? nextSlugWithId(row.changeSlug, tenantId)()
+            : row.changeSlug;
+
+          const changeLink = urlsForRouter(
+            router,
+            urlObjectForSlug(router, changeSlug)
+          );
```

Comment:
Let's extend this method to accept an object of query parameters, so we don't have to deal with query parameters ad-hoc.

Reply:
I've updated this - for some reason I haven't had to modify `urlsForRouter`, after inspecting the query inside it, it seems to be passing it around correctly, unless I'm misunderstanding how it's supposed to...

Reply:
No, I could believe that.